src/actions/sessionActions.js

```diff
@@ -74,7 +74,13 @@ export const fetchLoggedUser = () => async dispatch => {
 };
 
 export const registerUser = credentials => (dispatch) => {
-  return register(credentials).then((res) => {
-    // return dispatch(checkCredentials(res));
-  })
+  return register(credentials)
+    .then((res) => {
+      // return dispatch(checkCredentials(res));
+    })
+    .catch(err => {
+      const error = _get(err, 'response.data', err && err.response);
+      dispatch(checkCredentialsFailed(error));
+      // throw error;
```

Comment:
Please remove this comment unless you think is necessary for some reason.

Reply:
Deleted.